app/graphql/types/search_result/global_scope/target_type.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Types::SearchResult::GlobalScope::TargetType = GraphQL::UnionType.define do
```

Comment:
Well I mean, the `target` types kind of mirror the `categories`. I chose `target` because  the `entry` points to, well, the target :)

I think a type named `SpecificGlobalSearchEntry` might be confusing, since we already have the `GlobalSearchEntry`, so maybe something else entirely?